app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -74,9 +74,17 @@ function DocumentCaptureReviewIssues({
           ]}
```

Comment:
There are three potential issues with this work (that I am aware of, anyway):

1. When you try to open the SDK on a desktop / laptop, the SDK detects your device and will default to open the file picker.
    - @dawei-nava is currently [working on a PR](https://github.com/18F/identity-idp/pull/10112) that would prevent a user from getting to the desktop upload screen if selfie is enabled.
    - @charleyf pointed out there may be other times we're unaware of that this happens (like perhaps a very old phone or one without a camera)
2. One of the AC for this story is `not display/allow adding any images native camera`. I tried this out locally and was rather surprised to see that the native camera never launches - but it does fire the analytics event saying it does.
    - I will admit I lost track of this AC and haven't spent the time to fully research why this is happening. I thought I'd put a PR out for now while I research that / respond to comments, and perhaps fulfilling that AC will be a separate PR.
3. I think once Dawei's linked PR above is merged, there will need to be a bit of additional work to use the flag introduced in his PR to allow upload when in local dev or other envs where that flag is set to true.